numojo/core/array_manipulation_routines.mojo

```diff
@@ -0,0 +1,22 @@
+"""
+# ===----------------------------------------------------------------------=== #
+# ARRAY MANIPULATION ROUTINES
+# Last updated: 2024-06-16
+# ===----------------------------------------------------------------------=== #
+"""
+
```

Comment:
Do we still want this if most of it is in the NDArray Struct

Reply:
For the moment it seems not necessary. Even we want to make a function instead of a method of `NDArray`, we can do this as a static method.